code/modules/clothing/clothing.dm

```diff
@@ -792,3 +792,11 @@ BLIND     // can't see anything
 		deconstruct(FALSE)
 	else
 		..()
+
+/obj/item/clothing/neck
+	name = "necklace"
+	icon = 'icons/obj/clothing/neck.dmi'
+	body_parts_covered = UPPER_TORSO
+	slot_flags = SLOT_NECK
+	strip_delay = 40
+
```

Comment:
Very very nitpicky, but there's two newlines after the `strip_delay` rather than just one.
```suggestion

/obj/item/clothing/neck
	name = "necklace"
	icon = 'icons/obj/clothing/neck.dmi'
	body_parts_covered = UPPER_TORSO
	slot_flags = SLOT_NECK
	strip_delay = 40
```